src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(number) {
+      this.result += number;
+
+      return this;
```

Comment:
what happens if we remove this line?

Reply:
✕ 'operate' should support a chain call (1ms)
    ✕ 'operate' should correctly work in a long chain

i have 2 error after remove

Reply:
I think you remove `return this` in `operate` method because I don't have these errors. 

![image](https://github.com/mate-academy/js_advanced_calculator/assets/96138503/be70ca12-bd1c-4f36-9e0d-e05840bdf355)

Reply:
👀

Reply:
<img width="1056" alt="Снимок экрана 2023-08-09 в 09 20 50" src="https://github.com/mate-academy/js_advanced_calculator/assets/78561740/3a3c3ac4-78eb-40bb-8800-67c519d8b05e">

Reply:
if i delete s have problem with some task

is you remember, last time i have problem with some tasks test. I don*t know why